JUnit5-README.md

```diff
@@ -0,0 +1,193 @@
+# JUni5 extension
+
+Sunstone library comes with JUnit5 Sunstone extension for simpler testing in cloud environments.
+
+The extension is able to control Cloud resources on test class level. The module allows you to deploy resources in several ways: 
+- Azure template
+- AWS CloudFormation template
+- (TODO) JCloud resources providing property file depending on cloud:
+  - [azure-arm-README](azure-arm-README.md) 
+  - [baremetal-README](baremetal-README.md) 
+  - [ec2-README](ec2-README.md) 
+  - [docker-README](docker-README.md) 
+  - [openstack-README](openstack-README.md) 
+  - [wildfly-README](wildfly-README.md)
+
+## Test development
+
+This section shows a simple way how use Sunstone extension do deploy to Cloud using various technologies.
```

Comment:
From the learning curve perspective - I would suggest adding a few sentences about why this provider/module is different the other ones and why is it useful. I.e. uses native stack upload technologies instead of 3rd party java client, allows you to deploy resources beyond what JClouds allows, etc.